_rules/iframe-not-focusable-has-no-interactive-content-akn7bn.md

```diff
@@ -59,54 +63,54 @@ By setting the `tabindex` [attribute value][] of an `iframe` element to `-1` or
 
 #### Passed Example 1
 
-This `iframe` element contains no content that is part of [sequential focus navigation][].
+This `iframe` element does not have a `tabindex` [attribute value][] that is a negative number
 
 ```html
-<iframe tabindex="-1" srcdoc="<h1>Hello world</h1>"></iframe>
+<iframe tabindex="0" srcdoc="<a href='/'>Home</a>"></iframe>
 ```
 
-#### Passed Example 2
+### Failed
 
-This `iframe` element contains a link that is not part of [sequential focus navigation][] because of its `tabindex`.
+#### Failed Example 1
+
+This `iframe` element contains a [visible][] link that is part of its [flattened tabindex-ordered focus navigation scope][]. Note that the link is not part of the document's [sequential focus navigation][] because the containing `iframe` is excluded from the document's [tabindex-ordered focus navigation scope](https://html.spec.whatwg.org/multipage/interaction.html#tabindex-ordered-focus-navigation-scope) due to its negative `tabindex`.
 
 ```html
-<iframe tabindex="-1" srcdoc="<a href='/' tabindex='-1'>Home</a>"></iframe>
+<iframe tabindex="-1" srcdoc="<a href='/'>Home</a>"></iframe>
 ```
 
-#### Passed Example 3
+### Inapplicable
 
-This `iframe` element contains no [visible][] content because of the small size of the iframe.
+#### Inapplicable Example 1
+
+This `iframe` element contains no content that is part of [sequential focus navigation][].
```

Comment:
```suggestion
This `iframe` element contains no content that is part of its [flattened tabindex-ordered focus navigation scope][].
```
(in Failed Example 1, the link is part of the FTOFNS (!) but not of the sequential focus order, due to the negative tabindex of the iframe; the difference is that here there is nothing in th FTOFNS of the iframe, so we don't care about the negative tabindex).